composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt

```diff
@@ -68,7 +72,9 @@ private fun ObjectGrid(
     LazyVerticalGrid(
         columns = GridCells.Adaptive(180.dp),
         modifier = modifier.fillMaxSize(),
-        contentPadding = PaddingValues(8.dp)
+        contentPadding = WindowInsets.systemBars
+            .add(WindowInsets(left = 8.dp, right = 8.dp))
+            .asPaddingValues()
```

Comment:
WindowInsets.safeDrawing.asPaddingValues()

Reply:
Your code:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/086d5141-8204-4efb-b1ea-0bd4e3a180f4">

My code:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/96570fe2-98ae-4048-a376-b5a1b502202a">

Reply:
I have definitely not looked at landscape, my bad. But even with this code, I often get the content's right edge clipped both on a real Pixel device with a cutout and on an emulator.

<img width="1380" alt="image" src="https://github.com/user-attachments/assets/5ea85b59-28d8-4d4d-aaba-660747efff7b">

Reply:
Or, rotated the other way, content is drawn below the cutout still

<img width="1056" alt="image" src="https://github.com/user-attachments/assets/33299448-c47b-4f58-b100-168e2f361516">